### ISSUE TITLE
QA: /careers

### ISSUE THREAD

**Issue by venturewise** [hlxsites/merative#174]
@sachinmesh 's QA comments and screenshots are in the attached.
[Careers page Design QA v1.docx](https://github.com/hlxsites/merative/files/11193734/Careers.page.Design.QA.v1.docx)

https://main--merative2--hlxsites.hlx.page/careers

**Comment by helms-charity** [comment]
@sachinmesh can you check now between the circle overview list section and the Offices section? Should these have the 160px between each section, with the rainbow divider?
https://main--merative2--hlxsites.hlx.page/careers
![image](https://user-images.githubusercontent.com/3883795/231776576-6273cec3-78f1-4a56-9459-16b57b22ad77.png)

**Comment by helms-charity** [comment]
@keith-kaplan, @sachinmesh are we good to close this ticket?

**Comment by keith-kaplan** [comment]
@venturewise  I had one quick question question for the careers page. On the live page we have this text at the bottom - 
<img width="943" alt="image" src="https://github.com/hlxsites/merative/assets/111377779/f37ee2b3-9bbe-4e9d-a936-495953ef9b71">
https://www.merative.com/careers


What is the best way to replicate that? This is what I have so far...
<img width="1160" alt="image" src="https://github.com/hlxsites/merative/assets/111377779/171e948e-2a67-4225-a8cc-0c512a352891">
https://main--merative2--hlxsites.hlx.page/careers

**Comment by helms-charity** [comment]
Do you expect all text in the CTA area will be centered?
 If so, I can make that change by making the higher level div "cta-container" to be text-align: center.

![image](https://github.com/hlxsites/merative/assets/3883795/0d28a113-e231-48a0-a0da-53204a4a43f3)

**Comment by keith-kaplan** [comment]
Yeah, I think that can work. If that's done then I can move the copy into the CTA block.

**Comment by helms-charity** [comment]
Ok give it a try by moving it, see if that looks as expected.
